### ISSUE TITLE
If there is a Pre tag, the text from that term is not included in the HTML export

### ISSUE THREAD

**Issue by ryancasburn-KAI** [spencermountain/compromise#1073]
https://github.com/spencermountain/compromise/blob/4ef66b3e5798c63f3f0f3b7935ffae1597b6dd3b/src/1-one/output/api/html.js#L67

Test case: 
```javascript
  doc = nlp(`if i can recall, my grey dog loves pizza crusts (they are really good).`)
  html = doc.html({ '.red': 'my grey dog', '.blue': doc.match('loves') })
  t.equal(
    html,
    `if i can recall, <span class="red">my grey dog</span> <span class="blue">loves</span> pizza crusts (they are really good).`,
    here + 'html two classes'
  )
```

Currently returning:
```html
if i can recall, <span class="red">my grey dog</span> <span class="redTwo">loves</span> pizza crusts ( are really good).
```

suggestion is:
```javascript
out += t.pre || ''
out += t.text || ''
```

Any issues with that?

**Comment by spencermountain** [comment]
oh my gosh, it's missing the word??
yeah! please add a fix! 
holy cow

**Comment by spencermountain** [comment]
released as `14.11.1` - thank you!
